lib/rails_critical_css_server/rewrite.rb

```diff
@@ -6,9 +6,8 @@ class Rewrite
 
     delegate :request, :params, to: :template
 
-    def initialize(template, manifest: template.asset_url('application.css'), version: Config.version, skip: false, cache: true, &block)
```

Comment:
true. but then you have the manifest name twice in your template and it doesn't handle the generated hash in the filename.